nerfstudio/models/splatfacto.py

```diff
@@ -186,6 +186,29 @@ def __init__(
     ):
         self.seed_points = seed_points
         super().__init__(*args, **kwargs)
+        self.selected_gaussian_indices = []
```

Comment:
Let's define these inside `populate_submodules` next to the buttons/viewer control so all the new code for initializing floater removal is in one place